src/utils.h

```diff
@@ -10,3 +10,10 @@ void rv_gettimeofday(struct timeval *tv);
 
 /* Retrieve the value used by a clock which is specified by clock_id. */
 void rv_clock_gettime(struct timespec *tp);
+
+/* Detect the instruction is RV32C or not */
```

Comment:
Revise the comments and reference to Linux kernel source file for hash function.